packages/instituciones/src/useCases/db/planteles/delete.planteles.use-cases.js

```diff
@@ -0,0 +1,8 @@
+const deletePlantel = (deleteQuery) => async (identifierObject) => {
+  const { institucionId, plantelId } = identifierObject;
+
+  const plantelDeleted = await deleteQuery({ id: plantelId, institucionId });
```

Comment:
I made it in this way because the endpoint request in the next way
"../instituciones/institucionId/planteles/plantelId" to ensure that the data is correct.